pom.xml

```diff
@@ -137,30 +138,27 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.apache.kafka</groupId>
+        <artifactId>kafka_${scala.major.version}</artifactId>
+        <classifier>test</classifier>
+        <version>${kafka.version}</version>
+      </dependency>
       <dependency>
         <groupId>org.apache.kafka</groupId>
         <artifactId>kafka-clients</artifactId>
         <version>${kafka.version}</version>
       </dependency>
       <dependency>
         <groupId>org.apache.kafka</groupId>
-        <artifactId>connect-api</artifactId>
+        <artifactId>kafka-clients</artifactId>
+        <classifier>test</classifier>
         <version>${kafka.version}</version>
       </dependency>
       <dependency>
-        <groupId>org.apache.zookeeper</groupId>
-        <artifactId>zookeeper</artifactId>
-        <version>${zookeeper.version}</version>
-        <exclusions>
-          <exclusion>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
-          </exclusion>
-          <exclusion>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-          </exclusion>
-        </exclusions>
+        <groupId>org.apache.kafka</groupId>
+        <artifactId>connect-api</artifactId>
+        <version>${kafka.version}</version>
       </dependency>
       <dependency>
         <groupId>com.101tec</groupId>
```

Comment:
I think this zkclient dependency is probably obsolete now.

Reply:
Removed: https://github.com/matt-garlock/common-kafka/commit/2c3e1f609b8dde350039702791b313fd77e1a972#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8L163